mne/viz/utils.py

```diff
@@ -67,6 +67,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
 
     try:  # see https://github.com/matplotlib/matplotlib/issues/2654
         fig.canvas.draw()
+	fig.tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad)
         fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
```

Comment:
@Eric89GXL we need t this call, otherwise you will get silly errors. set_tight_layout will eventually apply the state, the problem is that tight_layout won't fail in agg mode.